components/src/core/components/List/List.vue

```diff
@@ -88,7 +88,9 @@
             </template>
             <template v-slot:option="{data}">
```

Comment:
As we discussed use the highlighted text passed from option slot of autocomplete. and add the sanitiser at autocomplete.
https://github.com/orangehrm/oxd/blob/ent/components/src/core/components/Input/Autocomplete/AutocompleteInput.vue#L47

Reply:
Updated as per discussion